docs/spring-boot-runtime/master.adoc

```diff
@@ -82,6 +82,14 @@ include::topics/assembly_cache-mission-booster.adoc[leveloffset=+2]
 :parameter-runtime-anchor!:
 
 
+// MESSAGING WORK QUEUE
+:context: messaging-work-queue
+:parameter-runtime: spring-boot
```

Comment:
Same as Node.js -- the way you set these attributes is highly error-prone, and this is a nice evidence of that.